apps/seroi/src/constant/laundry/SVCOrderAttributes.ts

```diff
@@ -0,0 +1,48 @@
+
+import { Attribute } from '@docs/ui/src/types/list.type';
+import attributes from '../attributes';
+
+const SVCOrder: Attribute[] = [
+  { ...attributes.type, description: 'Doc의 Type입니다. "ORDER"로 고정 값 입니다.' },
+  attributes.id,
+  { ...attributes.bucketId, description: '해당 주문이 발생한 함의 bucketId입니다.(단지)' },
+  { ...attributes.parentId, description: '해당 주문이 발생한 함의 docId입니다.(세탁함)' },
+  { name: 'title', required: true, type: 'string', description: '고객 휴대폰 번호 \n ex) 01012345678' },
+  attributes.deleted,
+  attributes.published,
+  attributes.updated,
+  attributes.created,
+  {
+    ...attributes.status,
+    type: '"DEPOSIT" | "PICKEDUP" | "SHIPPING" | "SHIPPING_CANCEL" | "INSPECT" | "CLEANING" | "PREPARE" | "DELIVERING" | "WAITPAY" | "PAYCOMPLETED" | "COMPLETED"',
```

Comment:
P2:
type에 긴 내용을 작성하니 UI 레이아웃이 문제가 되네요.
마침 아래에 module로 작성해주신것을 확인했는데, 위 내용에는 OrderStatus로만 명시해주셔도 좋겠습니다.

```suggestion
    type: 'OrderStatus',
```

![image](https://github.com/thepsyentist/docs/assets/39726717/6e5ffba0-e7e4-404e-91c4-2d79b31805f5)